### PR TITLE
Ability to disable pen tablet

### PR DESCRIPTION
**Pull request by mortifiedtux**
This PR adds `tablet` SConst option that allows it to disable by appending `tablet=no` in command line or build profile

### CODE REVIEW

**Comment by akien-mga** [comment]
Could you explain why you want this option?

**Comment by bruvzg** [comment]
What's the reason for disabling it completly? While some `wintab` drivers can be buggy, `winink` is standard Windows API and should not cause any issues.

**Comment by programneer** [comment]
> Could you explain why you want this option?

First, i don't have pen tablet, second, it's kinda useless for most games

**Comment by AThousandShips** [comment]
There are many different input modes that can be enabled or disabled, we don't have an option to disable controller support for example?

File: .github/workflows/linux_builds.yml [hunk 1]
```diff
@@ -80,7 +80,7 @@ jobs:
           - name: Minimal template (target=template_release, everything disabled)
             cache-name: linux-template-minimal
             target: template_release
-            sconsflags: modules_enabled_by_default=no disable_3d=yes disable_advanced_gui=yes deprecated=no minizip=no
+            sconsflags: modules_enabled_by_default=no disable_3d=yes disable_advanced_gui=yes deprecated=no tablet=no minizip=no
```
Comment (bruvzg) [inline .github/workflows/linux_builds.yml:83]: It's a Linux template build, but the flag is only effective on Windows, so it won't have nay effect.

**Pull request closed without merging**